### PR TITLE
Clarification about 'docker build --build-arg'

### PR DESCRIPTION
**Pull request by nshalman**
See #22860

### CODE REVIEW

File: docs/reference/commandline/build.md [hunk 1]
```diff
@@ -298,6 +298,9 @@ accessed like regular environment variables in the `RUN` instruction of the
 Dockerfile. Also, these values don't persist in the intermediate or final images
 like `ENV` values do.
 
+Using this flag will not alter the output you see when the `ENV` lines from the
```
Comment (duglin) [inline docs/reference/commandline/build.md]: `ENV` or `ARG` ?  I thought you were worried that the `ARG` output would be confusing.

**Comment by nshalman** [comment]
`ARG`, thank you. Let me fix this.

**Comment by justincormack** [comment]
LGTM, thanks for this. The commit is signed now...

**Comment by cpuguy83** [comment]
ping @thaJeztah

**Comment by vdemeester** [comment]
LGTM on the content, but I don't see the sign-off on the commit :sweat:

**Comment by thaJeztah** [comment]
LGTM as well, after signed-off

**Comment by nshalman** [comment]
Fixed.

**Comment by vdemeester** [comment]
Thanks @nshalman

**Pull request merged**